exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -231,16 +232,17 @@ func determineAwsOrigin(resource pdata.Resource) string {
 		}
 	}
 
-	// TODO(willarmiros): Only use platform for origin resolution once detectors for all AWS environments are
-	// implemented for robustness
 	if is, present := resource.Attributes().Get(conventions.AttributeCloudPlatform); present {
 		switch is.StringVal() {
+		// TODO: Replace with semantic convention const when aws_app_runner is included in an official semconv release
```

Comment:
Please open an issue to track this

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6210